StarVMC/StarAgmlLib/AgPosition.cxx

```diff
@@ -35,7 +36,12 @@ void AgMLDbMatrix::operator()( const char* table, const int row, double matrix[4
   TString tablename = table;
   tablename.ReplaceAll(" ","");
 
-  TTable *ttable = (TTable *)chain-> GetDataBase( tablename );
+#if ROOT_VERSION_CODE < ROOT_VERSION(6,00,0) 
```

Comment:
Disables the code which loads the ROOT/CINT files which hold the misalignment tables.  We will need to re-enable this code for production,.  This may require some work to make sure the loading / unloading of the ROOT/CINT files works.  TBD.